src/main.rs

```diff
@@ -1,6 +1,7 @@
 #[macro_use]
 extern crate tracing;
 
+use exitcode;
```

Comment:
this can be dropped, in rust 2018 you can reference a crate directly via its root name.